lib/src/modules/compiler/template_parser/GRAMMAR.md

```diff
@@ -0,0 +1,38 @@
+#Template Grammar
+
+###Todos
+* Can you interpolate comments?
+* Can you have multiple pipes?
```

Comment:
Meaning can you `A | pipe | pipe`? I actually don't know.

I sent an email to Victor on NgJs - let's see what they say.